### PR TITLE
(635) - Apply - Date of offence

### PR DESCRIPTION
**Pull request by richpjames**
# Changes in this PR
~~This builds on #299~~ merged
This adds a 'Date of offence' screen to the Apply form in the pattern established in #142 

### After
Apply -- shows a tasklist

### CODE REVIEW

**Review by pezholio — APPROVED** [review]
LGTM - just one suggestion about making the checkbox code easier to grok.

File: server/utils/utils.ts [hunk 1]
```diff
@@ -41,6 +43,12 @@ const kebabCase = (string: string) =>
     .replace(/[\s_]+/g, '-')
     .toLowerCase()
 
+/**
+ * Converts a string from any case to Sentence case
+ * @param string string to be converted.
+ * @returns name converted to sentence case.
+ */
+export const sentenceCase = (string: string) => Case.sentence(string)
```
[the remark this answers is not in the document]
Comment (richpjames) [reply server/utils/utils.ts:51]: The coverage checker seems to be happy? it runs in CI

File: server/form-pages/apply/risk-management-features/dateOfOffence.ts [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  }
+
+  renderTableHead() {
+    return ['Offence', 'Current', 'Previous'].map(headerItem => this.textValue(headerItem))
+  }
+
+  renderTableRow(offence: string) {
+    return [
+      this.textValue(sentenceCase(offence)),
+      ...['current', 'previous'].map(date =>
```
[the remark this answers is not in the document]
Comment (richpjames) [reply server/form-pages/apply/risk-management-features/dateOfOffence.ts:82]: Yeah good shout

File: server/utils/utils.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (pezholio) [reply server/utils/utils.ts:51]: Yeah, let's leave it in that case. It's a pretty simple function :)

**Pull request merged**